### ISSUE TITLE
Error when running activate.sh

### ISSUE THREAD

**Issue by petereichinger** [julianpoy/RecipeSage-selfhost#57]
Hey,

thanks for the amazing tool :) 

When I try to run the activate.sh script I get the following error message

```
{
  id: 'call-import-namespace',
  location: {
    column: 6,
    file: '/app/packages/backend/src/services/subscriptions.js',
    length: 6,
    line: 54,
    lineText: '    ? moment(new Date("1980"))',
    namespace: '',
    suggestion: ''
  },
  notes: [
    {
      location: [Object],
      text: 'Consider changing "moment" to a default import instead:'
    }
  ],
  pluginName: '',
  text: `Calling "moment" will crash at run-time because it's an import namespace object, not a function`
}
Database connection established
TypeError: moment is not a function
    at Object.extend (/app/packages/backend/src/services/subscriptions.js:110:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at run (/app/packages/backend/src/activate.js:22:3)
```

Any idea what is going on here?

**Comment by julianpoy** [comment]
Hi there! Thanks for the bug report. I've pushed a fix in fc35535c856d54bd68617f79a1a96fc21ce74a09.

Please pull down a new copy and try running activate.sh again.

**Comment by petereichinger** [comment]
Hey Julian, thank you for the fix :)
